keywords/subclassof.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+module.exports = function defFunc(ajv, customConstructors, overrideBuffer = false) {
+  var CONSTRUCTORS = {
+    Object: Object,
+    Array: Array,
+    Function: Function,
+    Number: Number,
+    String: String,
+    Date: Date,
+    RegExp: RegExp,
+    Error: Error
+  };
+
```

Comment:
instanceof also adds Buffer is it's available - probably should be here as well

Reply:
Yup, it was related to the fact that subclassof was validating against instances too. Buffer is a final class, and cannot be extended. But now that I'll modify this to be orthogonal to instanceof, it won't be a problem anymore.